tests/virtual_machines_test.go

```diff
@@ -298,13 +279,13 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		})
 		Context("When trying to create a VM after all MAC addresses in range have been occupied", func() {
 			PIt("[test_id:2162]should return an error because no MAC address is available", func() {
-				err := initKubemacpoolParams("02:00:00:00:00:00", "02:00:00:00:00:01")
-				Expect(err).ToNot(HaveOccurred())
+				//err := initKubemacpoolParams("02:00:00:00:00:00", "02:00:00:00:00:01")
```

Comment:
Is there a reason that those lines are commented or can we remove them?

Reply:
I saved them until the last commit and them removed them.